src/Monolog/Handler/ErrorLogHandler.php

```diff
@@ -71,13 +71,14 @@ protected function getDefaultFormatter(): FormatterInterface
      */
     protected function write(array $record)
     {
-        if ($this->expandNewlines) {
-            $lines = preg_split('{[\r\n]+}', (string) $record['formatted']);
-            foreach ($lines as $line) {
-                error_log($line, $this->messageType);
-            }
-        } else {
+        if (!$this->expandNewlines) {
```

Comment:
Here's how it would look the other way:

```php
protected function write(array $record)
    {
        if ($this->expandNewlines) {
            $lines = preg_split('{[\r\n]+}', (string) $record['formatted']);
            foreach ($lines as $line) {. // Nested foreach
               error_log($line, $this->messageType);
            }
            return;
        } 
        error_log((string) $record['formatted'], $this->messageType);
    }
```

You'll notice all the code is nested inside an if block.  In addition we now have a nested foreach block also nested.